ppod.py

```diff
@@ -18,8 +19,16 @@ def lambda_handler(event: dict, context: object) -> dict:
         logger.info(
             "Sentry DSN found, exceptions will be sent to Sentry with env=%s", env
         )
-    file_count = 0
+
     bucket = os.environ["BUCKET"]
+    ssm_client = client("ssm", region_name="us-east-1")
+    stream = ssm_client.get_parameter(
+        Name="/apps/ppod/stream-name", WithDecryption=True
```

Comment:
This isn't an encrypted parameter so we don't need the `WithDecryption` argument.